src/hypothesis/strategies.py

```diff
@@ -945,11 +955,26 @@ def builds(target, *args, **kwargs):
     the target.
 
     """
+    if target_and_args:
+        target, args = target_and_args[0], target_and_args[1:]
```

Comment:
I would write this `target, *args = target_and_args` - same effect and considerably shorter.

Reply:
Sadly, doesn't work in Python2. Maybe in 2020? :)

Reply:
Can't come soon enough 😭